### ISSUE TITLE
Allow to set force option of dcmread

### ISSUE THREAD

**Issue by medihack** [ImagingDataCommons/dicomweb-client#89]
Currently `dcmread` is used without the[ `force`](https://pydicom.github.io/pydicom/dev/reference/generated/pydicom.filereader.dcmread.html#pydicom-filereader-dcmread) option in [dicomweb-client](https://github.com/ImagingDataCommons/dicomweb-client/blob/e83fe98b7c0b3438222bd8a95592ab1d9e0e52d7/src/dicomweb_client/web.py#L1018). It would be nice to make this configurable to read also maybe not so valid DICOM files.

**Comment by pieper** [comment]
I don't see any downside to being more tolerant when reading.  It might make sense to try reading first, and if an exception is raised issue a warning and try again with the `force` option.

Anyone else have suggestions?  @hackermd

**Comment by medihack** [comment]
I am unsure if this would be a good solution as it would try to read each dataset twice when it is not valid, which could be quite a performance hit when reading many invalid datasets. But maybe it would be a good default (with some warning when the reading initially fails) but still have some option to do a forced reading initially.

**Comment by pieper** [comment]
From the documentation for `force`, it says it only checks for a metaheader, which I would think is very quick.  But we could check if it's an issue.

**Comment by medihack** [comment]
You are right. It then sounds like a good solution. Maybe a little warning on the console could still be printed.
